server/contexts/ctxerr/statistics_test.go

```diff
@@ -0,0 +1,48 @@
+package ctxerr
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestAggregate(t *testing.T) {
+	t.Run("returns an error if can't find a handler in the context", func(t *testing.T) {
+		_, err := Aggregate(context.Background())
+		require.Error(t, err)
+	})
+
+	t.Run("returns an error if it can't decode one of the errors stored", func(t *testing.T) {
+		eh := MockHandler{}
+		eh.RetrieveImpl = func(flush bool) ([]*StoredError, error) {
+			return []*StoredError{{Error: []byte("invalid")}}, nil
+		}
+		ctx := NewContext(context.Background(), eh)
+		_, err := Aggregate(ctx)
+		require.Error(t, err)
+	})
+
+	t.Run("returns an aggregation of the errors stored", func(t *testing.T) {
+		eh := MockHandler{}
+		eh.RetrieveImpl = func(flush bool) ([]*StoredError, error) {
+			return []*StoredError{
+				{Count: 10, Error: []byte(`{"cause": {"stack": ["a", "b", "c", "d"]}}`)},
+				{Count: 20, Error: []byte(`{"cause": {"stack": ["x", "y"]}}`)},
```

Comment:
We should add some cases with wraps to test the loop and maxTraceLen limit across multiple stacks.